### DISCUSSION TITLE
今天中午实验的时候出现购买人信息获取失败的问题

### DISCUSSION THREAD

**Discussion by Duruiui** [mikumifa/biliTickerBuy#316] [提问专区] [answered]
<img width="642" alt="image" src="https://github.com/mikumifa/biliTickerBuy/assets/139968189/0a174285-7334-4039-8518-b8f1b2ddb5cb">
但是json文件我是按教程配置好的，不知道为啥

**Comment by mikumifa** [comment]
重新登录一下吧，可能是登录信息有问题

**Comment by mikumifa** [comment]
第一次见到这个报错

**Reply by Duruiui** [reply]
<img width="644" alt="image" src="https://github.com/mikumifa/biliTickerBuy/assets/139968189/11b6b589-d28d-4129-a649-e0235e1a4a40">
很神奇的是我用其他漫展试验是好的，中午试着抢BW的时候就出现故障了

**Reply by CinnabarCHU** [reply, marked as answer]
大概率是可能不小心改了json的内容，可以重新配一下json试试……或者过一会试一下看看？感觉可能是服务器压力太大了，2XX的response应该是服务器还在处理信息吧……我今天手抢的时候客户端遇到几次卡住半天没相应的情况，这也是之前没遇到的，或许本质和你的情况类似。

**Reply by Duruiui** [reply]
后续感觉是json配置的问题导致的，我老老实实重新配了以后没有再出现过这样的问题了
